cfme/tests/containers/test_project_label.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.project import Project
+from utils import testgen, version
+from cfme.web_ui import CheckboxTable, toolbar as tb
+from utils.appliance.implementations.ui import navigate_to
+from cfme.containers.provider import ContainersProvider
+from utils import conf
+import time
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda provider: version.current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(2)]
+pytest_generate_tests = testgen.generate(
+    [ContainersProvider], scope='function')
+
+# CMP-10287
+
+
+def test_create_label_check(ssh_client, provider):
+    """ This test creates a new project in Openshift and labels it,
+        then it verifies the label exists and is visible on the summary
+        page.There is a sufficient time provided so that CFME is updated
+        with the latest data
+    """
+
+    hostname = conf.cfme_data.get('management_systems', {})[provider.key] \
+        .get('hostname', [])
+    username, password = provider.credentials['token'].principal, \
+        provider.credentials['token'].secret
+
+    ssh_client = ssh_client(
+        hostname=hostname,
+        username=username,
+        password=password)
```

Comment:
@gshefer We still need the credentials here, otherwise we are doing ssh to the CFME appliance.